CHANGELOG.md

```diff
@@ -23,11 +23,14 @@
   - `AZ_LOG_IOT_RETRY`
   - `AZ_LOG_IOT_SAS_TOKEN`
   - `AZ_LOG_IOT_AZURERTOS`
+- Removed `AZ_LOG_END_OF_LIST` log classification and `az_log_set_classifications()` from `az_log.h`.
```

Comment:
I would remove the "from az_log.h" part.

Reply:
Why? I prefer keeping it as it helps users know how they will be impacted and which header is being changed.